src/features/courses/course-route.ts

```diff
@@ -603,6 +603,57 @@ router.post('/question',
         }
     }));
 
+router.get('/question/:id/raw',
+    authenticationMiddleware,
+    validate(getQuestionRawValidation),
+    // This is a typescript workaround since it tries to use the type extractMap
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    asyncHandler(async (req: RederlyExpressRequest<any, unknown, GetQuestionRawRequest.body, unknown>, _res: Response, next: NextFunction) => {
+        if (_.isNil(req.session)) {
+            throw new Error(Constants.ErrorMessage.NIL_SESSION_MESSAGE);
```

Comment:
This is probably how it was because the Exceptions have been built out over time, but I think this should be RederlyError